WcaOnRails/db/migrate/20170517213035_update_list_of_countries.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class UpdateListOfCountries < ActiveRecord::Migration[5.0]
+  def up
+    ActiveRecord::Base.transaction do
+      Country.delete_all
+      Country::ALL_STATES.each do |state|
+        Country.create!(state)
+      end
+      # These substitions have been found by running the migration and checking
+      # users with iso2 not matching anything in the 'Country' table.
```

Comment:
Will you double check this after merging up this PR? Is it possible that in the mean time new people have signed up with `iso2`s not covered here?